Jenkinsfile

```diff
@@ -49,8 +49,17 @@ pipeline {
         stage('RSpec') {
           steps { sh 'ci/test rspec' }
         }
-        stage('Authenticators') {
-          steps { sh 'ci/test cucumber_authenticators' }
+        stage('Authenticators Config') {
+          steps { sh 'ci/test cucumber_authenticators_config' }
+        }
+        stage('Authenticators Status') {
+          steps { sh 'ci/test cucumber_authenticators_status' }
```

Comment:
Both related to OIDC i would change the name of the state for OIDC Authenticators..., and the feature file name as well..

Reply:
Why is `Authenticators Config` OIDC related? this test suite include `available_authn.feature` & `authn_config.feature` which have nothing to do with OIDC.

`Authenticators Status` _is_ related to OIDC but is actually separate. As you can see it doesn't assume that an OIDC provider is in tact. This test checks all the non-specific status checks (e.g webservice is loaded) so it runs separately. Please take a look at `authn_status.feature` to see this.

Let me know if you still think it should run under OIDC.

Reply:
agree for `Authenticators Config`. 
In `authn_status.feature `i see the last 2 tests contain:
When I GET "/**authn-oidc/keycloak/**non-existing-account/status" in several tests so this one should be oidc.
so lets change the Status to OIDC status (or moving the specific tests to different place)

Reply:
Again - the separation here is that these tests checks the general checks. We use here the OIDC authenticator just because it is the only authenticator that implements this check. The tests in `authn_status_oidc.feature` tests the oidc specific tests. The tests in `authn_status.feature` tests the general tests. 

Here's a rule of thumb - If once `authn-azure` is implemented we can replace every occurrence of `authn-oidc` to `authn-azure` and the test is still the same then that test belongs in `authn_status.feature`. If the replacements changed the test then it is oidc specific.

Reply:
Take a look here - https://github.com/cyberark/conjur/blob/master/app/domain/authentication/validate_status.rb

all these checks are not authenticator-specific. `authn_status.feature` tests this file. The only reason that feature file has the word OIDC is because if we wouldn't have had it then the check will fail [here](https://github.com/cyberark/conjur/blob/master/app/domain/authentication/validate_status.rb#L29) . If the authenticator doesn't exist or doesn't implement the status check then we can't test the case that the webservice doesn't exist

Reply:
And we will need to duplicate these tests in every `authn_status_something.feature` instead of having it in one place that doesn't assume anything about the authenticator. 

You can see that `authn_Status.feature` runs without even having a `keycloak` container up.